processing/src/main/java/org/apache/druid/segment/serde/DictionaryEncodedColumnPartSerde.java

```diff
@@ -294,7 +294,7 @@ public Deserializer getDeserializer()
     return new Deserializer()
     {
       @Override
-      public void read(ByteBuffer buffer, ColumnBuilder builder, ColumnConfig columnConfig)
+      public void read(String columnName, ByteBuffer buffer, ColumnBuilder builder, ColumnConfig columnConfig)
```

Comment:
Once we remove the argument from `ComplexMetricSerde` we don't need it here anymore either.